engines/sword1/control.h

```diff
@@ -159,13 +211,47 @@ class Control {
 	Sound *_sound;
 	Screen *_screen;
 	Logic *_logic;
-	uint8 *_font, *_redFont;
 	uint8 *_screenBuf;
 	Common::KeyState _keyPressed;
-	void delay(uint32 msecs);
+
 	Common::Point _mouseCoord;
 	uint16 _mouseState;
-	bool _mouseDown;
+
+	int _oldSnrStatus = SNR_BLANK;
+	bool _newPal = false;
+	Sprite *_slabs[SAVEBUTTONS - 6];
+	int32 _scrollIndex[2] = { 0, 0 };
+	int32 _speedFlag = 0;
+
+	int32 _currentButton = 0;
+	int32 _buttonPressed = 0;
+	int32 _buttonHold = 0;
+	int32 _slabSelected = 0;
+	int32 _firstDescription = 0;
+	char  _fileDescriptions[MAXSAVEGAMES][40];
```

Comment:
I'd recommend `byte` here. Some chars on some platforms are signed

Reply:
Thanks, I didn't know that; I'll change it!